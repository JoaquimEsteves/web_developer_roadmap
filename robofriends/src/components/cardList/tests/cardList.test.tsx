import { mount } from 'enzyme';
import CardList from '../cardList';
import '../../../setupTests';

it('try out the snapshot yo', () => {
    const example = [{
        id: 1, name: "John Doe",
        email: "johndoe@email.com", username: "johnDoe"
    }];
    expect(mount(CardList(example)).debug()).toMatchSnapshot();
})