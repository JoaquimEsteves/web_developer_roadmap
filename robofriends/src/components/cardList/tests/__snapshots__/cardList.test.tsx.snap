// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`try out the snapshot yo 1`] = `
"<div id=\\"cardList\\">
  <div className=\\"card\\">
    <img src=\\"https://robohash.org/1?size=200x200\\" alt=\\"card\\" />
    <div>
      <h2>
        John Doe
      </h2>
      <p>
        johndoe@email.com
      </p>
    </div>
  </div>
</div>"
`;
